src/assets/js/gfpdf-backbone.js

```diff
@@ -451,12 +451,12 @@
          * so the user isn't confused with the live-update display of the Font Name field.
          */
         this.modelBinder.bind(this.model, this.el, {
-            font_name: [{selector: '[name=font_name]'}, {selector: '[name=usage]', converter: this.model.cssDeclaration}],
```

Comment:
This code formatting (quite a lot of it in this file) isn't part of this PR. Please revert back to the original.

Reply:
Fixed - I just reverted back the old spacing.